aspnetcore/diagnostics/asp0028.md

```diff
@@ -31,36 +37,32 @@ This usage will be reported with a diagnostic message:
 
 ## Rule description
 
-The recommended way is to setup Kestrel to listen on `IPv6Any`.
+The recommended way to configure Kestrel to listen for incoming connections on all available `IPv6` network interfaces is with `IPv6Any`.
 
 ## How to fix violations
 
-For the reported code
-```csharp
-.UseKestrel().ConfigureKestrel(options =>
-{ 
-    options.Listen(IPAddress.Any, ...);
-})
-```
+For the problematic code, replace `Any` with `IPv6Any`:
 
-One can either explicitly change usage to `IPv6Any`:
-```csharp
+```diff
 .UseKestrel().ConfigureKestrel(options =>
 { 
-    options.Listen(IPAddress.IPv6Any, ...);
+-   options.Listen(IPAddress.Any, ...);
++   options.Listen(IPAddress.IPv6Any, ...);
 })
 ```
 
-or use another invocation - `options.ListenAnyIP()` without specifying any argument explicitly:
-```csharp
+Alternatively, use the [ListenAnyIP](https://source.dot.net/#Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerOptions.cs,1c84a7db2c1f6892) method without specifying any arguments:
```

Comment:
we have changed the [naming here](https://github.com/dotnet/aspnetcore/pull/58872#discussion_r1838509868) a bit - now `ListenAnyIP` is the suggested approach. Let's move it in order as the first option.